resources/views/emails/newpassword.blade.php

```diff
@@ -0,0 +1,4 @@
+@component('mail::message')
+<p>Dear {{ $user_name }}</p>
+<p>Thank you for joining Prisma. Your password is {{ $password }}</p>
```

Comment:
Beste {{ $user_name }}
Bedankt om deel uit te maken van Prisma. Uw wachtwoord is {{  $password }}